packages/textlint/src/createLinter.ts

```diff
@@ -113,13 +128,30 @@ export const createLinter = (options: CreateLinterOptions) => {
                 return kernel.fixText(fileContent, kernelOptions);
             });
         },
+        /**
+         * Lint text and fix it
+         * Note: fixText does not respect ignore file
+         * You can detect the file path is ignored or not by `isFilePathIgnored()`
+         * @param text
+         * @param filePath
+         */
         async fixText(text: string, filePath: string): Promise<TextlintFixResult> {
             const kernelOptions = {
                 ext: path.extname(filePath),
                 filePath,
                 ...baseOptions
             };
             return kernel.fixText(text, kernelOptions);
+        },
+        /**
+         * Check the file path is ignored or not
+         * Return true if the file path is ignored by `.textlintignore` or `ignoreFilePath` option
+         * @param filePath
+         */
+        async isFilePathIgnored(filePath: string): Promise<boolean> {
+            return isFilePathIgnored(filePath, {
+                ignoreFilePath: options.ignoreFilePath
+            });
         }
```

Comment:
Maybe, `scanFilePath(filePath): Prommise<{ status: "ignored" | "found" | "not-found" }>` is suitable.

Current implementation, `not-found.xxx` always return `true`.
It is a bit strange.